code/modules/mob/mob_helpers.dm

```diff
@@ -345,6 +345,33 @@
 			span_notice("You fix some of the [brute_damage ? "dents on" : "burnt wires in"] [human == user ? "your" : "[human]'s"] [affecting.name]."))
 		return TRUE //successful heal
 
+/// Heals a peg limb on a mob (based on item_heal_robotic)
+/proc/item_heal_peg(mob/living/carbon/human/human, mob/user, brute_heal, burn_heal)
```

Comment:
This doesn't need to be a global proc, could easily be on `/human` (or on the bodypart itself IG)

Reply:
its right next to the robo version of the same thing 😭

Reply:
That's also cringe

Reply:
Done!